File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/FindPatientInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class FindPatientInfo
    {
        private String _patientCode;
        private String _patientName;
        private String _patientName2;
        private String _sex;
        private String _birth;
        private String _lastCommingDate;
        private String _address;
        private String _ipwonYn;
        private String _treatmentArea;
        private String _tel;

        public FindPatientInfo()
        {
        }

        public FindPatientInfo(string patientCode, string patientName, string patientName2, string sex, string birth, string lastCommingDate, string address, string ipwonYn, string treatmentArea, string tel)
        {
            _patientCode = patientCode;
            _patientName = patientName;
            _patientName2 = patientName2;
            _sex = sex;
            _birth = birth;
            _lastCommingDate = lastCommingDate;
            _address = address;
            _ipwonYn = ipwonYn;
            _treatmentArea = treatmentArea;
            _tel = tel;
        }

        public string PatientCode
        {
            get { return _patientCode; }
            set { _patientCode = value; }
        }

        public string PatientName
        {
            get { return _patientName; }
            set { _patientName = value; }
        }

        public string PatientName2
        {
            get { return _patientName2; }
            set { _patientName2 = value; }
        }

        public string Sex
        {
            get { return _sex; }
            set { _sex = value; }
        }

        public string Birth
        {
            get { return _birth; }
            set { _birth = value; }
        }

        public string LastCommingDate
        {
            get { return _lastCommingDate; }
            set { _lastCommingDate = value; }
        }

        public string Address
        {
            get { return _address; }
            set { _address = value; }
        }

        public string IpwonYn
        {
            get { return _ipwonYn; }
            set { _ipwonYn = value; }
        }

        public string TreatmentArea
        {
            get { return _treatmentArea; }
            set { _treatmentArea = value; }
        }

        public string Tel
        {
            get { return _tel; }
            set { _tel = value; }
        }
    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/FindPatientInfo.java
package nta.med.data.model.ihis.system;

import java.sql.Timestamp;

public class FindPatientInfo {
	private String patientCode;
    private String patientName;
    private String patientName2;
    private String sex;
    private Timestamp birth;
    private Timestamp lastCommingDate;
    private String address;
    private String ipwonYn;
    private String treatmentArea;
    private String tel;
	
	public FindPatientInfo(String patientCode, String patientName,
			String patientName2, String sex, Timestamp birth,
			Timestamp lastCommingDate, String address, String ipwonYn,
			String treatmentArea, String tel) {
		super();
		this.patientCode = patientCode;
		this.patientName = patientName;
		this.patientName2 = patientName2;
		this.sex = sex;
		this.birth = birth;
		this.lastCommingDate = lastCommingDate;
		this.address = address;
		this.ipwonYn = ipwonYn;
		this.treatmentArea = treatmentArea;
		this.tel = tel;
	}
	public String getPatientCode() {
		return patientCode;
	}
	public void setPatientCode(String patientCode) {
		this.patientCode = patientCode;
	}
	public String getPatientName() {
		return patientName;
	}
	public void setPatientName(String patientName) {
		this.patientName = patientName;
	}
	public String getPatientName2() {
		return patientName2;
	}
	public void setPatientName2(String patientName2) {
		this.patientName2 = patientName2;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public Timestamp getBirth() {
		return birth;
	}
	public void setBirth(Timestamp birth) {
		this.birth = birth;
	}
	public Timestamp getLastCommingDate() {
		return lastCommingDate;
	}
	public void setLastCommingDate(Timestamp lastCommingDate) {
		this.lastCommingDate = lastCommingDate;
	}
	public String getIpwonYn() {
		return ipwonYn;
	}
	public void setIpwonYn(String ipwonYn) {
		this.ipwonYn = ipwonYn;
	}
	public String getTreatmentArea() {
		return treatmentArea;
	}
	public void setTreatmentArea(String treatmentArea) {
		this.treatmentArea = treatmentArea;
	}
	public String getTel() {
		return tel;
	}
	public void setTel(String tel) {
		this.tel = tel;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	
}
